### DISCUSSION TITLE
nnn-vim display icons

### DISCUSSION THREAD

**Discussion by ldelossa** [mcchrish/nnn.vim#110] [Q&A] [answered]
Hello, 

I compiled nnn to display icons. I have a NerdFont patch'd font configured for my terminal. I don't see icons in nnn-vim. Is this expected?

**Comment by mcchrish** [comment, marked as answer]
Can you try if you can see the icons when running nnn in a terminal buffer? i.e. Run `:terminal` then run nnn in it. Or just `:terminal nnn`

**Reply by ldelossa** [reply]
Yup, I see them when I run `nnn` in a terminal buffer.

**Reply by mcchrish** [reply]
I'm not really sure what's going on. How about if you run nnn in picker mode `nnn -p -`?

**Reply by ldelossa** [reply]
No luck

**Reply by luukvbaal** [reply]
Are you by chance using the `-C` flag in your `g:nnn#command`?
Otherwise I wouldn't know either.

**Reply by ldelossa** [reply]
Looks like i am. 

https://github.com/ldelossa/dotfiles/blob/master/nvim/nnn.vim

**Reply by luukvbaal** [reply]
Well there's your problem, icons won't work in 8 bit mode.
Remove the `-C` flag and you'll get icons.

**Reply by ldelossa** [reply]
Thanks! That did it.
